### PR TITLE
Minor tweaks to OpenAPI 3.0 additions in the README

### PR DESCRIPTION
**Pull request by brandur**
Mainly these changes:

* Rephrase `OpenAPI2` to `OpenAPI 2`.
* Rephrase `OpenAPI3` to `OpenAPI 3`.
* Reworded a few places for grammar/etc.
* I've alphabetized a few lists of options in some places.
* Added some missing punctuation.
* Corrected a few instances of spelling.

@ota42y I figured it was easier to send a pull request than to point all of these out in a review. Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,47 +256,41 @@ describe Committee::Middleware::Stub do
...
 ```
 
-If you want to select manualy, please pass 'openapi_parser' object to committee.
-This gem added gem dependency so you can use always
+If you want to select the type manually, pass an `OpenAPI 3` object to the `schema` option manually:
 
 ```ruby
 open_api = OpenAPIParser.parse(YAML.load_file('open_api_3/schema.yaml'))
-schema = Committee::Drivers::OpenAPI3.new.parse(open_api)
+schema = Committee::Drivers::OpenAPI 3.new.parse(open_api)
```
Comment (ota42y) [inline README.md]: This isn't valid ruby code.
So please fix correct class name `OpenAPI3`.

File: README.md [hunk 2]
```diff
@@ -306,52 +304,57 @@ use Committee::Middleware::RequestValidation, schema: 'json string'
...
 hash = JSON.load(json_path)
 
-if hash['openapi']&.start_with?('3.') # OpenAPI3 specification require this key and version
-  return Committee::Drivers::OpenAPI3.new.parse(hash)
-elsif hash['swagger'] == '2.0' # OpenAPI2 require swagger key
-  return Committee::Drivers::OpenAPI2.new.parse(hash)
-else 
+# OpenAPI 3 requires the `openapi` key and a version
+if hash['openapi']&.start_with?('3.')
+  return Committee::Drivers::OpenAPI 3.new.parse(hash)
```
Comment (ota42y) [inline README.md]: Please fix correct class name

File: README.md [hunk 3]
```diff
@@ -306,52 +304,57 @@ use Committee::Middleware::RequestValidation, schema: 'json string'
...
+
+# OpenAPI 2 requires the `swagger` key
+elsif hash['swagger'] == '2.0'
+  return Committee::Drivers::OpenAPI 2.new.parse(hash)
```
Comment (ota42y) [inline README.md]: Please fix correct class name

**Comment by brandur** [comment]
Thanks @ota42y! That's what I get for doing a last minute find and replace. Code is now fixed. Can you take another look?

**Comment by ota42y** [comment]
Thanks!!!! 👍 👍 👍

**Pull request merged**